src/Raven.Studio/wwwroot/App/views/shell.html

```diff
@@ -1,4 +1,4 @@
-<div class="layout-container" data-bind="css: { 'collapse-menu': collapseMenu(), 'pin-notifications': notificationCenter.pinNotifications(), 'show-notifications': notificationCenter.showNotifications() }">
+<div class="layout-container" data-bind="css: { 'collapse-menu': collapseMenu() || smallScreen(), 'pin-notifications': notificationCenter.pinNotifications(), 'show-notifications': notificationCenter.showNotifications(), 'show-search': searchBox.showSearchSmallScreen() }">
```

Comment:
I don't see declaration of: showSearchSmallScreen anywhere. Did you forgot to commit all files?

Reply:
I didn't stage it. Fixed it now.